ent/schema/tracks.go

```diff
@@ -0,0 +1,31 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/schema/field"
+	"github.com/albe2669/spotify-viewer/ent/schema/pulid"
+)
+
+type Track struct {
+	ent.Schema
+}
+
+func (Track) Mixin() []ent.Mixin {
+	return []ent.Mixin{
+		AtTimeMixin{},
+		pulid.MixinWithPrefix("TR"),
+	}
+}
+
+func (Track) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("track_id").NotEmpty().GoType(pulid.ID("")),
```

Comment:
Should track_id possibly just be a string where we store Spotify's id for it, then we can check that such that we don't accidentally upload the same track twice?

Reply:
I would agree with doing it like that since I assume Spotify uses unique IDs on their end. Would simplify things a lot for us. Would probably write this into the issue about connecting the other data struct to the database as we need to make sure the IDs match in that case for now this is fine with just having the one table 👍